mern-ecommerce/controllers/auth.js

```diff
@@ -62,7 +62,7 @@ exports.requireSignin = expressJwt({
   userProperty: 'auth',
 });
 
-exports.isAuth = (req, res, next) => {
+exports.isAuth = (request, response, next) => {
```

Comment:
Here is the review:

In the `isAuth` function, the parameter `request` is used as `req` in the function body. It should be consistent throughout the function. 

The `isAuth` function does not handle errors that might occur during the authentication check. It would be better to add error handling to provide a proper error response when an error occurs.

The function uses `req.profile` and `req.auth` without checking if they exist. It would be better to add checks to ensure they are not null or undefined before using them.

The function does not check if `req.profile._id` and `req.auth._id` are of the same type before comparing them. It would be better to add a type check to ensure they are of the same type.

The error message 'Access denied' is not very informative. It would be better to provide a more detailed error message to give users a better understanding of what went wrong.